json-schema-to-html/schema.js

```diff
@@ -130,19 +130,19 @@ function renderItemsValidation(schema, type, octothorpes) {
 	var text = []
 	var validationItems = []
 
-	var content = schema.type == "array" ? "elements" : "value";
+	var content = schema.type == "array" ? "Each element" : "The value";
```

Comment:
The comparison has to be against `type` and not `schema.type`.

Reply:
Ooops, did the change locally, but failed to commit it :face_in_clouds:.